### PR TITLE
Warn that we don't have SDK for the FE

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Webpack 5 makes our FE build fail

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
~~- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.~~

### CODE REVIEW

File: docs/js/guides/sdk-in-browser.mdx [hunk 1]
```diff
@@ -11,6 +11,13 @@ keywords:
   - sap cloud sdk
 ---
 
+:::warn
+
+The latest Webpack update introed braking changes that make the building part of this guide fail. You can try to resolve them yourself or wait before we test and release an update to this guide. We can't provide a specific ETA for such update.
```
Comment (artemkovalyov) [inline docs/js/guides/sdk-in-browser.mdx:16]:
```suggestion
:::warn

The latest Webpack update introed braking changes that make the building part of this guide fail.
You can try to resolve them yourself or wait before we test and release an update to this guide.
We can't provide a specific ETA for such an update.
```

File: docs/js/guides/sdk-in-browser.mdx [hunk 2]
```diff
@@ -11,6 +11,15 @@ keywords:
   - sap cloud sdk
 ---
 
+:::warn
+
+The latest Webpack update introed braking changes that make the building part of this guide fail.
```
Comment (artemkovalyov) [inline docs/js/guides/sdk-in-browser.mdx:16]:
```suggestion
The latest Webpack update introduced braking changes that make the building part of this guide fail.
```

File: styles/Vocab/SAP/accept.txt [hunk 3]
```diff
@@ -68,3 +68,4 @@ autogenerate
 [Ww]eb[Ff]lux
 [Dd]ockerfile
 [Mm]iddleware
+[Ww]ebpack
```
Comment (artemkovalyov) [inline styles/Vocab/SAP/accept.txt:71]:
```suggestion
```

File: docs/js/guides/sdk-in-browser.mdx [hunk 4]
```diff
@@ -11,6 +11,15 @@ keywords:
   - sap cloud sdk
 ---
 
+:::warn
```
Comment (artemkovalyov) [inline docs/js/guides/sdk-in-browser.mdx:14]:
```suggestion
:::caution
```

**Comment by FrankEssenberger** [comment]
I am a bit confused. Is this the issue related to XSSEC introduction? Then webpack is not the issue.

**Pull request closed without merging**